src/common.rs

```diff
@@ -784,13 +790,56 @@ impl PartialEq<FQNameRef<'_>> for FQName {
 pub enum Identifier {
     /// This variant is case sensitive
     /// "fOo""bAr""" is stored as fOo"bAr"
-    Quoted(String),
+    Quoted(IdentifierWithSpan),
     /// This variant is case insensitive
     /// Only ascii alphanumeric and _ characters are allowed in this variant
     /// fOo_bAr is stored as fOo_bAr
-    Unquoted(String),
+    Unquoted(IdentifierWithSpan),
+}
+
+#[derive(Debug, Clone, Eq, Ord, PartialOrd, PartialEq, Deserialize, Hash)]
+pub struct IdentifierWithSpan {
+    value: String,
```

Comment:
These fields should be public, otherwise the user wont be able to access the span or value.

Reply:
ok